test/e2e/app-dir/not-found/multi-root-layout/index.test.ts

```diff
@@ -15,8 +15,12 @@ describe('not-found-multi-root-layout', () => {
     expect(await browser.elementByCss('h1').text()).toBe('Sub Not Found')
   })
 
-  it('should render root not-found for uncaught routes', async () => {
-    const browser = await next.browser('/404')
-    expect(await browser.elementByCss('h1').text()).toBe('Root Not Found')
-  })
+  // turbo does not create root layout if missing, so skip this test
+  // GH#63103
+  if (!process.env.TURBOPACK) {
```

Comment:
Let's update the manifest here instead so we can track this needing updating

https://github.com/vercel/next.js/blob/canary/test/turbopack-build-tests-manifest.json